profiles/standard/standard.install

```diff
@@ -106,12 +106,28 @@ function standard_install() {
   theme_enable(array($default_theme));
   theme_disable(array('stark'));
 
-  // Set front page to "node".
-  config_set('system.core', 'site_frontpage', 'node');
+  // Add the home page layout.
+  $path = backdrop_get_path('profile', 'standard') . '/config/';
+  $contents = file_get_contents($path . 'layout.layout.home.json');
+  $active_directory = config_get_config_directory('active');
+  $active_storage = new ConfigFileStorage($active_directory);
+  try {
+    $config_data = $active_storage->decode($contents);
+    module_load_include('inc', 'config', 'config.sync');
+    config_sync_validate_file('layout.layout.home', 'create', NULL, $config_data);
+    config_sync_file('layout.layout.home', 'create', $config_data);
+    cache_clear_all();
 
-  // Insert default pre-defined node types into the database. For a complete
-  // list of available node type attributes, refer to the node type API
-  // documentation at: http://api.drupal.org/api/HEAD/function/hook_node_info.
+    // Set front page to "home".
+    config_set('system.core', 'site_frontpage', 'home');
+  }
+  catch (ConfigStorageException $e) {
+    $data = FALSE;
+  }
```

Comment:
Hm, I'd like to see why `config` isn't working for profiles. It should get run through `module_enable()` like all other modules AFAIK. I think we might need to update `config_install_default_config()` so that it works with things other than modules. It'd probably be simpler and better than manually attempting this in the profile code.